scripts/flash/index.js

```diff
@@ -19,10 +19,12 @@ var marketInfo = require("./market-info");
 var showInitialReporter = require("./show-initial-reporter");
 var fork = require("./fork");
 var approval = require("./approve-account");
+var listMarketOrders = require("./list-market-orders");
+var fillMarketOrders = require("./fill-market-orders");
 
-var commands = ["get-balance", "list-markets", "designate-report", "initial-report", "dispute-contribute", "finalize-market", "push-time", "market-info", "show-initial-reporter", "fork", "approval"];
+var commands = ["get-balance", "list-markets", "designate-report", "initial-report", "dispute-contribute", "finalize-market", "push-time", "market-info", "show-initial-reporter", "fork", "approval", "list-market-orders", "fill-market-orders"];
 var NETWORKS = ["aura", "clique", "environment", "rinkeby", "ropsten"];
-var methods = [getBalance, listMarkets, designatedReport, initialReport, disputeContribute, finalizeMarket, pushTime, marketInfo, showInitialReporter, fork, approval];
```

Comment:
These two lists are closely related, given how large they have become, it's hard to read. Might be worth converting it to an object which maps command to method (or if they could be the exact same thing, that would be great, too)